src/retasc/__main__.py

```diff
@@ -6,16 +6,23 @@
 from retasc import __version__
 from retasc.retasc_logging import init_logging
 from retasc.tracing import init_tracing
+from retasc.validator.validate_rules import validate_rule
 
 
 def parse_args():
     parser = argparse.ArgumentParser(description=doc)
     parser.add_argument(
         "-v", "--version", action="version", version=f"%(prog)s {__version__}"
     )
+    parser.add_argument(
+        "--validate-rule", type=str, help="Path to the rule file to validate"
```

Comment:
Better use [sub-commands](https://docs.python.org/3.12/library/argparse.html#sub-commands) (e.g. command line `retasc validate-rule`) and include also `generate-schema` as one.

Reply:
Done, much better now :)